### PR TITLE
AvalancheJS deprecation message

### PR DESCRIPTION
**Pull request by laviniat1996**

### CODE REVIEW

**Comment by meaghanfitzgerald** [comment]
Does this still need a review?

**Comment by laviniat1996** [comment]
> Does this still need a review?

Yes it does. It's been here for a while and I really want to merge it.

File: docs/apis/avalanchejs/README.md [hunk 1]
```diff
@@ -1,5 +1,15 @@
 # AvalancheJS Overview
 
+:::caution
+
+Please note that **AvalancheJS** is deprecated.
+[This](https://www.npmjs.com/package/@avalabs/avalanchejs) is the current **JavaScript Library**
```
[the remark this answers is not in the document]
Comment (laviniat1996) [reply docs/apis/avalanchejs/README.md:6]: You are right. I cannot seem to find the link we have now, can you pls help?

[the remark this answers is not in the document]
Comment (meaghanfitzgerald) [reply docs/apis/avalanchejs/README.md:6]: bumping this @cgcardona @laviniat1996

**Comment by meaghanfitzgerald** [comment]
@cgcardona @laviniat1996 this PR is over 2 months stale. Please let me know if you are actively working on it and need more time, otherwise, I will close it EOW.

**Pull request closed without merging**